evalml/tests/component_tests/test_label_encoder.py

```diff
@@ -221,3 +221,17 @@ def test_label_encoder_with_positive_label_with_custom_indices():
     y_with_custom_indices = pd.Series(["b", "a", "a"], index=[5, 6, 7])
     _, y_transformed = encoder.transform(None, y_with_custom_indices)
     assert_index_equal(y_with_custom_indices.index, y_transformed.index)
+
+
+def test_label_encoder_categorical_boolean_values():
```

Comment:
I would repeat the same advice for this test as I would for Becca's - what motivated the adding of this test case?  I am not sure the test name as-is reflects the "why" of why this test exists.  I think it's worthwhile to add a little context via a docstring to the test function...perhaps link it to a woodwork version upgrade.